core/domain/platform_parameter_registry.py

```diff
@@ -349,6 +349,57 @@ def _to_platform_parameter_model(
     platform_parameter_domain.DataTypes.STRING
 )
 
+Registry.create_platform_parameter(
```

Comment:
Note that, this is not introduction of new params but just moving them to parameter registry for a clear separation.